File: python/URI/1043.py
a,b,c=input().split()
a=float(a)
b=float(b)
c=float(c)

if a<b+c and abs(b-c)<a and b<a+c and abs(a-c)<b and c<a+b and abs(a-b)<c:
    print("Perimetro = {:.1f}".format(a+b+c))
else:
    area = ((a+b)*c)/2
    print("Area = {:.1f}".format(area))

File: C/URI/1043.c
#include <stdio.h>
void main(){
	float A,B,C,P, S;
	scanf("%f%f%f",&A,&B,&C);
	
	if (A<B+C && B<C+A && C<A+B){
		P = A+B+C;
		printf("Perimetro = %.1f\n", P);
	}
	else {
		S = ((A+B)*C)/2;
		printf("Area = %.1f\n", S);
	}
}